config/controller.yaml

```diff
@@ -57,6 +57,7 @@ spec:
           "-stderrthreshold", "INFO",
           "-el-image", "ko://github.com/tektoncd/triggers/cmd/eventlistenersink",
           "-el-port", "8080",
+          "-el-security-context", "true",
```

Comment:
@khrm i see default value for `el-security-context` is `true` only 
https://github.com/tektoncd/triggers/blob/main/pkg/reconciler/eventlistener/config.go#L25

Reply:
Yes, that's why I put it as true here. I added it here so that if a user want to configure, he can refer here release.yaml instead of code.